src/components/Field/SupportMeToField.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useContext } from 'react';
+import { Input, Row, Col, Button, Dropdown, Menu, Form, Switch, Tooltip } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAlignLeft, faExpandArrowsAlt } from '@fortawesome/free-solid-svg-icons';
+
+import { FieldProps } from '.';
+import { globalContext } from '../../context/GlobalContextProvider';
+import styles from '../../styles/fields.module.scss';
+
+const { TextArea } = Input;
+
+export enum STATS_ALIGNMENT {
+    LEFT = 'left',
+    CENTRE = 'center',
+    RIGHT = 'right',
+}
+
+export interface SupportMeOptions {
+    height?: string;
+    width?: string;
+    alignment?: STATS_ALIGNMENT;
+    fitImage?: boolean;
+}
+
+export interface SupportMeData {
+    paypal?: string;
+    buymeacoffee?: string;
+}
+
+export interface SupportMeProps extends FieldProps {
+    id?: string;
+    data?: SupportMeData;
+    options?: SupportMeOptions;
+}
+
+export const generateAlignmentTags = (alignment: STATS_ALIGNMENT, type: 'start' | 'end') => {
+    if (alignment && type === 'start') return `<div align="${alignment}">`;
+    else if (!alignment && type === 'start') return `<div>`;
+    else if (alignment && type === 'end') return `</div>`;
+    else return '';
+};
+
+export const generateImageTag = (data: SupportMeData, options: SupportMeOptions) => {
+    const statsUrl = {
+        paypal: `https://paypal.me/${data.paypal}`,
+        buymeacoffee: `https://www.buymeacoffee.com/${data.buymeacoffee}`,
+    };
+    let html = '';
+    if (data.paypal)
+        html += `
+            <a href="${statsUrl.paypal}" target="_blank" style="display: inline-block;">
+                <img
+                    src="https://img.shields.io/badge/Donate-PayPal-green.svg" 
+                    align="${options.alignment ? options.alignment : 'left'}"
+                />
+            </a>`;
+    if (data.buymeacoffee)
+        html += `
+            <a href="${statsUrl.buymeacoffee}" target="_blank" style="display: inline-block;">
+                <img
+                    src="https://img.shields.io/badge/Donate-Buy%20Me%20A%20Coffee-yellow.svg" 
```

Comment:
Please use `&style=flat-square` to match with other shield design
also, use the `orange` color of BuyMeACoffee to make it match the brand color.

Reply:
You missed the orange color instead of yellow.